src/main/java/cn/nukkit/event/player/PlayerChatEvent.java

```diff
@@ -21,25 +21,15 @@ public static HandlerList getHandlers() {
 
     protected Set<CommandSender> recipients = new HashSet<>();
 
-    public PlayerChatEvent(Player player, String message) {
-        this(player, message, "chat.type.text", null);
-    }
-
-    public PlayerChatEvent(Player player, String message, String format, Set<CommandSender> recipients) {
+    public void sendMessage(Player player, String message) {
         this.player = player;
         this.message = message;
+	this.format = "chat.type.text";
```

Comment:
I just changed constructor into method. I didn't change essence of codes.
It is similar with previous codes, and chat is work successfully!